doc/user/content/third-party/dbt.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Using dbt"
+description: "Get details about using Materialize with dbt"
+menu:
+  main:
+    parent: 'third-party'
+---
+
+You can use [dbt] to easily create views and materialized views in your `materialized`
+instance. To do so, all you need to do is install our [dbt-materialize] adapter.
```

Comment:
That dbt is neither capitalized nor treated like a program name feels odd to me––this is their decision, not ours, so nothing much needs to change. However, I think "dbt-materialize" should either be code-highlighted, or written as "dbt-Materialize". I have a slight preference for the former, but whatever you think is best.